.github/workflows/e2e-test.yml

```diff
@@ -13,7 +13,7 @@ on:
 jobs:
   e2e:
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 10
+    timeout-minutes: 20
```

Comment:
Added another commit as on Windows this test sometimes take a bit longer than 10m